### DISCUSSION TITLE
any reason to upgrade?

### DISCUSSION THREAD

**Discussion by hs6666** [pivpn/pivpn#1660] [Q&A]
Hi, I setup my pivpn on a Pi v1 around two years ago, happily using it since then

I occasionally run apt update && apt upgrade every now and again

Do I need to actually re-run the install script for any other security reasons?

Thanks

**Comment by coolapso** [comment]
Then everything is fine. as for PiVPN scripts, if all is working fine for you and you haven't felt any weird behavior or that you need to update the scripts then there is no reason. 

Most of updates that concern PiVPN itself go onto the installer script and very little ones go on the "management" scripts.

**Comment by hs6666** [comment]
Thanks, will leave as is, love this project but have to admit will be running the free version of tailscale soon in parallel, I'm not that techy and its so so easy to install/setup compared to dealing with openvpn files etc)
